src/Airship/Internal/Decision.hs

```diff
@@ -71,18 +69,8 @@ hIfNoneMatch = "If-None-Match"
 -- tree
 ------------------------------------------------------------------------------
 
-data FlowState m = FlowState
-    { _contentType :: Maybe (MediaType, Webmachine m ResponseBody) }
+type FlowStateT m = StateT () (Webmachine m)
```

Comment:
Note: I've left the, now redundant, `StateT` just to avoid having to removing all the `lift`s which would make this PR even more unreadable. Happy to do a second pass and remove it if/when this PR is merged (or just before if that's the preference).